pkg/app/piped/cmd/piped/piped.go

```diff
@@ -355,7 +358,11 @@ func (p *piped) run(ctx context.Context, input cli.Input) (runErr error) {
 	analysisResultStore := analysisresultstore.NewStore(apiClient, input.Logger)
 
 	// Create memory caches.
-	appManifestsCache := memorycache.NewTTLCache(ctx, time.Hour, time.Minute)
+	appManifestsCache, err := memorycache.NewLRUCache(150)
```

Comment:
```suggestion
	appManifestsCache, err := memorycache.NewLRUCache(p.appManifestCacheCount)
```

Reply:
This is to use the cache count from flag. 🙏

Reply:
Thanks. I commited the suggestion.